client/my-sites/domains/domain-management/list/domain-only.jsx

```diff
@@ -43,6 +44,7 @@ const DomainOnly = ( { primaryDomain, hasNotice, recordTracks, siteId, slug, tra
 
 	return (
 		<div>
+			<PageViewTracker path="/view/:site" title="Domain only" />
```

Comment:
I think we can figure out a better placement for this query component. Right now, this will only count a page view if we've got a primary domain, however, there is a case above where the user will hit a page view, but it will not be recorded. 

Historically we've gone around this problem by moving the `PageViewTracker` a layer above, in the corresponding controller logic.

Reply:
There shouldn't be a scenario where `primaryDomain` is not defined, I suspect the early `return` when `primaryDomain` isn't defined, likely only takes place when the state is still loading.

Reply:
Thanks for elaborating, however, my point is still valid. As far as I know, page views should be tracked as early as possible instead of waiting until all the data has loaded.